Service/Request/RequestFactory.php

```diff
@@ -55,19 +63,22 @@ class RequestFactory implements RequestFactoryInterface
      * @param RequestVisitorRegistryInterface $requestVisitorRegistry
      * @param UriFactory                      $uriFactory
      * @param MessageFactory                  $messageFactory
+     * @param bool                            $ignoreBodyOnGetRequests
      */
     public function __construct(
         EndpointRegistryInterface $endpointRegistry,
         SerializerInterface $serializer,
         RequestVisitorRegistryInterface $requestVisitorRegistry,
         UriFactory $uriFactory,
-        MessageFactory $messageFactory
+        MessageFactory $messageFactory,
+        bool $ignoreBodyOnGetRequests
```

Comment:
lets update the naming to take now in the account that its being used for more than just GET requests, preferably related to the name of the config item, so like `strictModeEnabled`

Reply:
Fixed